src/renderer/views/Watch/Watch.js

```diff
@@ -928,6 +929,11 @@ export default defineComponent({
       this.updateLastViewedPlaylist(payload)
     },
 
+    handleVideoReady: function () {
+      this.videoPlayerReady = true
```

Comment:
please reset this to false when the video changes (in the watch handler for $route)